e2etest/newe2e_resource_manager_azstorage.go

```diff
@@ -17,6 +22,112 @@ type AzureAccountResourceManager struct {
 	armClient *ARMStorageAccount
 }
 
+func (acct *AzureAccountResourceManager) ApplySAS(a Asserter, URI string, loc common.Location) string {
+	a.AssertNow("account must not be nil", Not{IsNil{}}, acct)
+	switch loc {
+	case common.ELocation.Blob():
+		skc, err := blob.NewSharedKeyCredential(acct.accountName, acct.accountKey)
+		a.NoError("Create shared key", err)
+
+		p, err := blobsas.AccountSignatureValues{ // todo: currently this generates a "god SAS", granting all permissions. This is bad! Let's not do this in prod.
```

Comment:
a thought for the future, maybe we make this configurable for the user to pass in a string that represents the sas permissions they want (ideally the user passes the minimal required) and we can parse it as a SignatureValues obj and pass it here

Reply:
Great idea! I was planning on addressing this alongside some stronger auth method selection for RunAzCopy.